Riot/Modules/Home/AllChats/AllChatsActionPanelView.xib

```diff
@@ -0,0 +1,49 @@
+        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="20020"/>
+        <capability name="documents saved in the Xcode 8 format" minToolsVersion="8.0"/>
+    </dependencies>
+    <objects>
+        <placeholder placeholderIdentifier="IBFilesOwner" id="-1" userLabel="File's Owner"/>
+        <placeholder placeholderIdentifier="IBFirstResponder" id="-2" customClass="UIResponder"/>
+        <visualEffectView opaque="NO" contentMode="scaleToFill" id="xHa-hQ-5Sc" customClass="AllChatsActionPanelView" customModule="Riot" customModuleProvider="target">
+            <rect key="frame" x="0.0" y="0.0" width="559" height="96"/>
+            <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMaxY="YES"/>
+            <view key="contentView" opaque="NO" clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="center" insetsLayoutMarginsFromSafeArea="NO" id="KsT-WA-OkE">
+                <rect key="frame" x="0.0" y="0.0" width="559" height="96"/>
+                <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+                <subviews>
+                    <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="system" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="iny-AL-dNx">
+                        <rect key="frame" x="501" y="4" width="46" height="31"/>
+                        <state key="normal" title="Button"/>
+                        <buttonConfiguration key="configuration" style="plain" image="square.and.pencil" catalog="system"/>
+                    </button>
+                    <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="system" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="i6Z-u3-V1Z">
+                        <rect key="frame" x="12" y="4" width="45" height="31"/>
+                        <state key="normal" title="Button"/>
+                        <buttonConfiguration key="configuration" style="plain" image="ellipsis.circle" catalog="system"/>
+                    </button>
+                </subviews>
+                <constraints>
+                    <constraint firstItem="i6Z-u3-V1Z" firstAttribute="top" secondItem="KsT-WA-OkE" secondAttribute="top" constant="4" id="3gn-bP-jjR"/>
+                    <constraint firstItem="i6Z-u3-V1Z" firstAttribute="leading" secondItem="KsT-WA-OkE" secondAttribute="leading" constant="12" id="kvL-cg-72f"/>
+                    <constraint firstAttribute="trailing" secondItem="iny-AL-dNx" secondAttribute="trailing" constant="12" id="qQn-ku-t8s"/>
+                    <constraint firstItem="iny-AL-dNx" firstAttribute="top" secondItem="KsT-WA-OkE" secondAttribute="top" constant="4" id="t0d-yD-QKf"/>
+                </constraints>
+            </view>
+            <blurEffect style="regular"/>
```

Comment:
Follow up thought: perhaps using a `UIToolbar` instead of a custom `UIView` would be most sensible here anyway. It would fix this, automatically handle sizing on devices without a safe area (or with large dynamic type enabled) and also do the pretty thing with `scrollEdgeAppearance` like we currently get on the tab bar.

Reply:
Definitely the right thing to do 🙏